t/09-terminal.t

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/perl
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+use 5.018;
+use warnings;
+use Carp qw( confess );
+use English qw( -no_match_vars );
+use POSIX qw( :sys_wait_h pause );
+use Socket qw( PF_UNIX SOCK_STREAM sockaddr_un );
+use Time::HiRes qw( usleep );
+
+use Test::More tests => 12;
```

Comment:
We stopped using counted tests and use `done_testing` instead at the end of tests.